lib/htty/cli/commands/path_set.rb

```diff
@@ -48,7 +48,11 @@ def self.see_also_commands
   def perform
     add_request_if_new do |request|
       self.class.notify_if_cookies_cleared request do
-        request.path_set(*escape_or_warn_of_escape_sequences(arguments))
+        path = arguments[0]
+        path_components = path.split('/')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.